components/suggest/src/db.rs

```diff
@@ -210,36 +210,65 @@ impl<'a> SuggestDao<'a> {
     pub fn fetch_amp_suggestions(&self, query: &SuggestionQuery) -> Result<Vec<Suggestion>> {
         let keyword_lowercased = &query.keyword.to_lowercase();
         let suggestions = self.conn.query_rows_and_then_cached(
-            "SELECT s.id, k.rank, s.title, s.url, s.provider, s.score
-             FROM suggestions s
-             JOIN keywords k ON k.suggestion_id = s.id
-             WHERE s.provider = :provider AND
-             k.keyword = :keyword",
-             named_params! {
+            r#"
+            SELECT
+              s.id,
+              k.rank,
+              s.title,
+              s.url,
+              s.provider,
+              s.score
+            FROM
+              suggestions s
+            JOIN
+              keywords k
+              ON k.suggestion_id = s.id
+            WHERE
+              s.provider = :provider
+            AND k.keyword = :keyword
```

Comment:
```suggestion
              s.provider = :provider
              AND k.keyword = :keyword
```

Did `./bqetl format` really align the `AND` with the `WHERE` in this query? 🤨 It looks like `AND` is indented under `WHERE` in all the other queries—let's match that indentation here, too, even if it's not what the tool did for this one?